packages/presentation/lib/app/app_bloc.dart

```diff
@@ -133,6 +143,13 @@ class _AppBloc extends BlocImpl<BaseArguments, AppData> implements AppBloc {
   BasePage? _currentPage() => state.pages.lastOrNull;
 
   void _updateData() {
+    final event = EventEntity(
+      AnalyticsEvents.appEvents.screenView,
+      {'payload': _currentPage()?.name},
+    );
+
+    logAnalyticsEventUseCase(event);
```

Comment:
let's create separate usecase for tracking screens. it will receive only screen name. it will allow us not to write screenView and payload all the time in bloc. we will write it only once in usecase

Reply:
@nikolay-ahramovich-idf  still actual

Reply:
Fixed.